setup/docker/initdb.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
Can't this be added as a simple shell script instead of a PHP file? I'd expect something like:
```
#!/bin/sh -e

test $# -eq 4 || exit 1

mysql --host="$1" --user="$2" --password="$3" --database="$4" setup/database/schema.sql
```

Reply:
I want to test connection, and I would like to add a test to skip the schema import when it already exists. Do we really want to do this inside a shell script?

Reply:
If yes, I will bring everything inside entrypoint.sh

Reply:
Skipping the schema import is a good idea, but I think this can be done in the SQL file itself (aka `IF NOT EXISTS`). If that's OK for you I can do the change on master branch directly.

Remaining part is the import, for which I think the dependency on PHP may not be needed, but that's not a big deal anyway :)

Reply:
Good idea! Let's do the skip in SQL.
At the end, a unique shell script makes it easier to understand and maintain.

Reply:
I realize that mysql client is not installed on php container, and it would be better to keep it this way.
I suggest:
- keep php script, it is not harmful